nanoFramework.Device.Bluetooth.nuspec

```diff
@@ -19,9 +19,10 @@
 This package requires a target with nanoFramework.Device.Bluetooth v$nativeVersion$ (checksum $checksum$).</description>
     <tags>nanoFramework C# csharp netmf netnf nanoFramework.Device.Bluetooth</tags>
     <dependencies>
-      <dependency id="nanoFramework.CoreLibrary" version="1.12.0-preview.18" />
-      <dependency id="nanoFramework.Runtime.Events" version="1.10.0-preview.11" />
-      <dependency id="nanoFramework.System.Text" version="1.1.3-preview.22" />
+      <dependency id="nanoFramework.CoreLibrary" version="1.12.0-preview.9" />
```

Comment:
All these versions are behind the current ones. Please update.

Reply:
Hard to keep up :-)